pdvega/_register.py

```diff
@@ -0,0 +1,9 @@
+from ._pandas_internals import (register_dataframe_accessor,
+                                register_series_accessor)
+
+from ._plotting import FrameVgPlotMethods, SeriesVgPlotMethods
+
+
+def register():
+    register_dataframe_accessor('vgplot')(FrameVgPlotMethods)
+    register_series_accessor('vgplot')(SeriesVgPlotMethods)
```

Comment:
Looking at these, it seems like they are meant to just decorate the classes when they are defined. I think it would be cleaner to do that and do away with the ``_register.py`` submodule all together.

Reply:
Yup, you're right. I've updated to reflect these changes.